Update boringssl to a0fb951d2a26a8ee746b52f3ba81ab011a0af778

yongtang opened the pull request:
This fix update boringssl to `a0fb951d2a26a8ee746b52f3ba81ab011a0af778`, which contains bug fix related to #13733.

See https://github.com/tensorflow/tensorflow/pull/13734#issuecomment-337440239 for update details.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

yongtang committed:
Update boringssl to a0fb951d2a26a8ee746b52f3ba81ab011a0af778

This fix update boringssl to a0fb951d2a26a8ee746b52f3ba81ab011a0af778,
which contains bug fix related to 13733.

See https://github.com/tensorflow/tensorflow/pull/13734#issuecomment-337440239
for update details.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

yongtang committed:
Remove unneeded patch part for boringssl, as 13733 has been fixed.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

tensorflow-jenkins commented:
Can one of the admins verify this patch?

vrv commented:
@tensorflow-jenkins test this please

davidben commented on third_party/boringssl/add_boringssl_s390x.patch:
```diff
@@ -1,17 +1,6 @@
---- a/src/include/openssl/base.h	2017-10-10 15:50:34.000000000 +0000
-+++ b/src/include/openssl/base.h	2017-10-15 19:49:38.182154627 +0000
+--- a/src/include/openssl/base.h	2017-10-17 21:30:48.000000000 +0000
++++ b/src/include/openssl/base.h	2017-10-18 04:07:23.590036926 +0000
```
Existing issue, but per the discussion here, you probably don't want to be patching this at all anymore.
https://github.com/tensorflow/tensorflow/pull/11170#issuecomment-318697044

yongtang replied:
Thanks @davidben for pointing out. The patch has been removed.

yongtang committed:
Remove s390x patch for boringssl

Per discussion:
https://github.com/tensorflow/tensorflow/pull/13798#discussion_r145459255

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

vrv commented:
@tensorflow-jenkins test this please

vrv commented:
@davidben let us know if this is okay to merge

davidben commented:
(I don't actually work on Tensorflow.)

vrv commented:
Yes, but I'm not sure if what your suggestion about removing the patch is what was done; I thought the idea was to remove the patch entirely, not to just undo the changes to the existing patch.

yongtang commented:
@vrv In the PR the `third_party/boringssl/add_boringssl_s390x.patch` file has been deleted.

In `tensorflow/workspace.bzl` file, the boringssl section has been changed from `patched_http_archive` -> `native.http_archive` so patching is gone.

Also, in case `s390x` support is desired in TensorFlow in the future, from the comment link provided (https://github.com/tensorflow/tensorflow/pull/11170#issuecomment-329252942) it is possible to disable the GCP support to handle the big-endian issue. I don't have any access to `s390x` so I might not be able to help in that case though.

vrv commented:
Ok great, thanks.  I didn't have the context.  This PR looks good, though I guess the patch file should just be deleted entirely.
